extensions/amp-story/1.0/amp-story-page.js

```diff
@@ -502,8 +494,7 @@ export class AmpStoryPage extends AMP.BaseElement {
         mediaEl.addEventListener('error', resolve, true /* useCapture */);
       });
     });
-
-    return Promise.all(mediaPromises);
+    return Promise.all(mediaPromises).then(() => this.markPageAsLoaded_());
```

Comment:
Last comment: I think with your refactoring, we're able to remove `markPageAsLoaded_()` entirely, which is awesome :+1:

Reply:
@newmuis are we using the `PAGE_LOADED` event that is dispatched from `markPageAsLoaded_`? We aren't sure, but if it's not used for analytics or anything then I think we can remove this method altogether.

Reply:
It's a public API, effectively, so we don't know who is depending on it (e.g. tools)

Reply:
Jon, what do you think of still removing it? I wouldn't consider it a public API since it's a JavaScript event, that is impossible to use unless you write invalid AMP (script). It's not documented anywhere, so I'd be surprised if anyone uses it.
If we do break someone, these people could switch to the `CommonSignals.LOAD_END` signal instead.

Reply:
@ithinkihaveacat FYI we are thinking of removing the `PAGE_LOADED / 'ampstory:pageload'` JS event, since we don't think anyone is using it, and if they are, they could use the workaround mentioned above.

We will be mentioning this in the slack channel and documenting the workaround somewhere (not sure where is the right place? @newmuis might have an idea).